README.md

```diff
@@ -1,5 +1,7 @@
+# source{d} Engine
```

Comment:
The image below is already a kind of fancy header with the exact same text. I think this is redundant.